dist/iso.js

```diff
@@ -10,6 +10,17 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
 
 var KEY_NAME = 'data-iso-key';
 
+var rLt = /</g;
+var rGt = />/g;
+var rLte = /&lt;/g;
+var rGte = /&gt;/g;
+var rEnc = /[<>]/;
+var rDec = /&lt;|&gt;/;
+
+var toString = function toString(val) {
```

Comment:
using `toString` like this is an unrecoverable runtime error in some older browsers - it'd be better to name it something different (something that's not a preexisting global or Object.prototype key).